proto/cosmos/tx/v1beta1/service.proto

```diff
@@ -40,6 +40,24 @@ service Service {
   rpc GetBlockWithTxs(GetBlockWithTxsRequest) returns (GetBlockWithTxsResponse) {
     option (google.api.http).get = "/cosmos/tx/v1beta1/txs/block/{height}";
   }
+  // TxDecode decodes the transaction.
+  //
+  // Since: cosmos-sdk 0.47
+  rpc TxDecode(TxDecodeRequest) returns (TxDecodeResponse) {
+    option (google.api.http) = {
+      post: "/cosmos/tx/v1beta1/decode"
+      body: "*"
```

Comment:
Curious, what does this mean?

Reply:
Its a HTTP->gRPC mapping, where we're mapping post method `/cosmos/tx/v1beta1/decode` with `TxDecode` rpc, in which `body: "*"` means to take all the  HTTP post data as input.